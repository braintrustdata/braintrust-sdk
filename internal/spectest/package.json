{
  "name": "@braintrust/spectest",
  "version": "0.0.1",
  "description": "SDK specification test runner for cross-language testing",
  "private": true,
  "type": "module",
  "scripts": {
    "build": "tsc runner.ts --esModuleInterop --resolveJsonModule --module nodenext --moduleResolution nodenext --target es2022",
    "runner": "node runner.js"
  },
  "dependencies": {
    "braintrust": "workspace:*",
    "openai": "^4.77.3",
    "yaml": "^2.6.1"
  },
  "devDependencies": {
    "@types/node": "^20.10.5",
    "tsx": "^4.19.2",
    "typescript": "^5.4.4"
  }
}
