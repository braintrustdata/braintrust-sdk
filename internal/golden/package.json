{
  "name": "golden",
  "version": "1.0.0",
  "description": "",
  "private": true,
  "main": "index.js",
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@braintrust/langchain-js": "workspace:*",
    "@langchain/anthropic": "^0.3.30",
    "@langchain/core": "^0.3.78",
    "@langchain/openai": "^0.3.17",
    "braintrust": "workspace:*",
    "openai": "^6.3.0"
  }
}
